api/tests/test_security_user.py

```diff
@@ -127,3 +127,14 @@ def test_security_user_api_endpoint_cant_allow_delete(self, mock_verify_token):
             HTTP_AUTHORIZATION="Token {}".format(self.token_admin),
         )
         self.assertEqual(response.data, {'detail': 'Method "DELETE" not allowed.'})
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_admin_can_filter_security_users_by_status(self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {'email': self.admin_user.email}
+        # url = reverse('security_users-list')
+        response = client.get(
+            '{}?active={}'.format(self.security_users_admin_url, self.security_user.active),
+            HTTP_AUTHORIZATION="Token {}".format(self.token_admin),
```

Comment:
For this test let's confirm `self.security_user` is in the response. also  confirm the user isn't in the response for a filter with `not self.security_user.active`

Reply:
Alright